frontend/src/pages/NewStudent.vue

```diff
@@ -187,19 +193,43 @@ export default {
 
 
     showAddLanguage() {
-        this.$buefy.dialog.prompt({
-          message: `Add new language`,
-          inputAttrs: {
-            placeholder: 'e.g. Italian',
-            maxlength: 255,
-          },
-          confirmText: 'Add',
-          onConfirm: (value) => {
-            this.languageData.push(value)
-            this.$refs.languageComplete.setSelected(value)
+      this.$buefy.dialog.prompt({
+        message: `Add new language`,
+        inputAttrs: {
+          placeholder: 'e.g. Italian',
+          maxlength: 255,
+        },
+        confirmText: 'Add',
+        onConfirm: async (value) => {
+          // POST to /api/nativeLanguages 
```

Comment:
This POST is now handled as part of the Student creation. What you should do here is just pass the newly-added native language in with the rest of the new student details.

Reply:
Yup, I've simplified the showAddLanguage method (it just assigns value to this.selected.NativeLanguage) in light of that